package.json

```diff
@@ -4,14 +4,15 @@
   "description": "Code Climate test reporter client for javascript projects",
   "main": "index.js",
   "scripts": {
-    "test": "mocha"
+    "test": "istanbul cover --report lcov _mocha"
```

Comment:
The need for `_mocha` here is odd but it is a known issue: https://github.com/gotwarlost/istanbul/issues/44